src/platform/plugins/shared/workflows_management/public/entities/workflows/model/use_workflow_stats.ts

```diff
@@ -16,6 +16,7 @@ export function useWorkflowStats() {
   const { http } = useKibana().services;
 
   return useQuery<WorkflowStatsDto>({
+    networkMode: 'always',
```

Comment:
why the `networkMode` change is needed?

Reply:
When I worked offline, worflow UI failed to work, because by default http client paused requests